themes/bootstrap3/js/common.js

```diff
@@ -66,19 +66,18 @@ var VuFind = (function VuFind() {
   };
 
   var initDisableSubmitOnClick = function initDisableSubmitOnClick() {
-    $('[data-disable-on-submit]').on('submit', function handleOnClickDisable() {
-      var $form = $(this);
-      // Disable submit elements via setTimeout so that the submit button value gets
-      // included in the submitted data before being disabled:
-      setTimeout(
-        function disableSubmit() {
-          $form.find('[type=submit]').prop('disabled', true);
-        },
-        0
-      );
-    });
+    var formSubmission = document.forms.item("data-disable-on-submit");
+    formSubmission.setAttribute(onsubmit, true);
```

Comment:
It looks like this may be the issue. Look into using `addEventListener('submit', fn)` as a replacement for `$.submit(fn)`. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#examples

Reply:
Updated and tested with the additional button on home[age, seems to be working fine. Please review again.